src/frontend/planner_test/tests/testdata/explain.yaml

```diff
@@ -169,3 +169,19 @@
     explain (logical) create table t1(v1 int);
   explain_output: |
     LogicalSource
+- sql: |
+    explain create table t (v1 int, v2 varchar);
+  explain_output: |
+    StreamMaterialize { columns: [v1, v2, _row_id(hidden)], pk_columns: [_row_id] }
+    └─StreamExchange { dist: HashShard(_row_id) }
+      └─StreamRowIdGen { row_id_index: 2 }
+        └─StreamDml { columns: [v1, v2, _row_id] }
+          └─StreamSource
```

Comment:
Just curious, why do we still have a stream source even if no connector is specified?

Reply:
Yes. It's only for receiving barriers.

Reply:
Maybe we can do https://github.com/risingwavelabs/risingwave/issues/7856. But it maybe does not have so much meaning.